wp-content/themes/core/pcss/utilities/mixins/_buttons.pcss

```diff
@@ -20,93 +20,67 @@
 }
 
 /* -----------------------------------------------------------------------------
- * Button - Solid
+ *
+ * Button - Repeated styles across all button components
+ *
  * ----------------------------------------------------------------------------- */
 
-@define-mixin button-solid {
-	background-color: var(--color-mine-shaft);
-	color: var(--color-white);
+@define-mixin button {
```

Comment:
Similar as what we were doing before this iteration, I think it might make the most sense, for simplicity, to either not have mixin's for now for buttons or each type of button should get it's own mixin with it's core styles (for reuse) vs. having one global button mixin used across all buttons.

Reply:
Takeaway from call, rename this mixin to `button-shared`